scss/civicrm/administer/settings/_connection.scss

```diff
@@ -0,0 +1,8 @@
+.crm-connection-block {
+  @include civicrm-table;
+
+  h3 {
+    @include panel-header;
+    margin-bottom: 0;
```

Comment:
why should the `h3` have no bottom margin only for this page? Why the exception?

Reply:
For all `h3` tags which are on top of page,
- If it's Inside the block, we make it look like panel-header
- If it's out side the block, we make it look like panel-header + make it's `margin-bottom: 0` to make it stick with it's nearest element. In general case, it's a crm-block.

So, this one falls under second scenario